pkg/webhooks/utils/event.go

```diff
@@ -30,16 +30,9 @@ func GenerateEvents(engineResponses []*response.EngineResponse, blocked bool) []
 					events = append(events, e)
 				}
 			}
-		} else {
-			if er.IsSkipped() {
-				for i := range er.PolicyResponse.Rules {
-					e := event.NewPolicySkippedEvent(event.AdmissionController, event.PolicySkipped, er, &er.PolicyResponse.Rules[i])
```

Comment:
Can we remove `NewPolicySkippedEvent` now ?

Reply:
Ah yes, I just checked and the function is not being used anywhere anymore.